src/main/kotlin/com/terraformation/backend/accelerator/ModuleEventService.kt

```diff
@@ -98,18 +96,17 @@ class ModuleEventService(
 
   fun updateEventStatusIfEventUpToDate(event: ModuleEventScheduledEvent, status: EventStatus) {
     systemUser.run {
-      val moduleEvent =
-          try {
-            eventStore.fetchOneById(event.eventId)
-          } catch (e: EventNotFoundException) {
-            log.error("Module event ${event.eventId} not found.")
-            return@run
-          }
-
-      if (moduleEvent.revision == event.revision) {
-        eventStore.updateEventStatus(event.eventId, status)
-      } else {
-        log.info("Module event ${event.eventId} has been changed. Not updating status.")
+      try {
+        val moduleEvent = eventStore.fetchOneById(event.eventId)
+        if (moduleEvent.revision == event.revision) {
+          eventStore.updateEventStatus(event.eventId, status)
+        } else {
+          log.info("Module event ${event.eventId} has been changed. Not updating status.")
+        }
+      } catch (e: EventNotFoundException) {
+        log.error("Module event ${event.eventId} not found.")
+      } catch (e: Exception) {
+        log.error("Update status for event ${event.eventId} failed: " + e.message)
```

Comment:
Logging the whole exception including stack trace would be useful here since this should only happen for unexpected errors. That is, pass `e` as an additional parameter instead of appending `e.message` to the log message.